packages/block-editor/src/hooks/test/gap.js

```diff
@@ -1,9 +1,31 @@
 /**
  * Internal dependencies
  */
-import { getGapCSSValue } from '../gap';
+import { getGapCSSValue, getGapBoxControlValueFromStyle } from '../gap';
 
 describe( 'gap', () => {
+	describe( 'getGapBoxControlValueFromStyle()', () => {
+		it( 'should return `null` if argument is falsey', () => {
+			expect( getGapBoxControlValueFromStyle( undefined ) ).toBeNull();
+			expect( getGapBoxControlValueFromStyle( '' ) ).toBeNull();
+		} );
+		it( 'should return box control value from string', () => {
+			const expectedValue = {
+				top: '88rem',
+				left: '88rem',
+			};
+			expect( getGapBoxControlValueFromStyle( '88rem' ) ).toEqual( expectedValue );
```

Comment:
Unfortunately Prettier is complaining about this line! (It wants it to be on a separate line)

```
  17:64  error  Replace `·expectedValue·` with `⏎↹↹↹↹expectedValue⏎↹↹↹`  prettier/prettier
```